CHANGES.md

```diff
@@ -5,10 +5,42 @@
 * Adjust classes to abide by naming conventions for attributes.
 * Change `ExponentiatedGradient` signature by renaming argument `T` to
   `max_iter`, `eta_mul` to `eta0`, and by adding `run_linprog_step`.
+* API refactoring to separate out different uses of `eps` within
+  `ExponentiatedGradient`. It is now solely responsible for setting a bound on
+  the lambda vectors. The other usage of `eps` as a bound for constraints is
```

Comment:
this description of the remaining use of `eps` is not quite correct. i'll do a direct edit after i finish my pass.

Reply:
Looks like you got to this based on commit messages.